src/glimpse.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * A tool to view the data structure coming from the server
+ *
+ * To run, for example, type
+ *
+ *     glimpse tcp://localhost:1234
+ *
+ * Author: Jun Zhu, zhujun981661@gmail.com
+ *
+ */
+#include "kb_client.hpp"
+
+#include <iostream>
+#include <thread>
+#include <chrono>
+
+
+int main (int argc, char* argv[]) {
+    std::string addr;
+    if (argc >= 2) addr = argv[1];
+    else throw std::invalid_argument("Server address required!");
+
+    karabo_bridge::Client client;
+
+    client.connect(addr);
+
+    while (true) {
```

Comment:
I would remove the loop. if it is to check the data structure, one request is enough. And it is dangerous too request continuously data as it can "steal" data from other tools...